### PR TITLE
Add Engine instrumentation!

### PR DESCRIPTION
**Pull request by daniman**
This PR adds Engine instrumentation to Githunt!

We will need to add an `ENGINE_API_KEY` to `.env` in order for engine data to start reporting.

### CODE REVIEW

File: api/engineConfig.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    {
+      name: 'standardCache',
+      epoch: 1,
+      timeout: '1s',
+    },
+  ],
+  operations: [
+    {
+      signature: `
+        query CurrentUserForLayout {
```
Comment (stubailo) [inline api/engineConfig.js]: I think this is going to get every user to see the same username when they log in.

**Comment by daniman** [comment]
Yeah, this query isn't the right one to be cached. I think we should be caching the Feed query with a shorter TTL.

We should fix the Hot and New tabs, and then we can cache the Top and Hot tabs with a shorter TTL (~1min? ~30sec? the `ttl` param's unit is in milliseconds) and leave the New tab uncached.

@peggyrayzis Can you take a look next time you're working on Githunt?

**Comment by stubailo** [comment]
Yeah I think there are a few small things we need to do:

1. Make the GitHunt React UI work properly, otherwise it's hard to test things out
2. Split up the feed query to not also query the current user
3. Configure engine to cache the feed query instead of this one, for a relatively short TTL

@peggyrayzis let me know if I can help.

**Comment by stubailo** [comment]
This is the query in question: https://github.com/apollographql/GitHunt-React/blob/5616bc978599c85b36df3a8da4c57076940b5aa9/ui/graphql/FeedQuery.graphql

**Comment by daniman** [comment]
We can also split up this PR by ignoring caching for now but leaving the rest of the Engine instrumentation here, and revisit caching later.

**Pull request merged**